build.sbt

```diff
@@ -22,20 +24,40 @@ lazy val root = (project in file("."))
     maintainer in Docker := "Lunatech Labs <lunatech@lunatech.com>",
     dockerBaseImage := "openjdk:8-jre-slim",
 
+    scalacOptions ++= Seq(
+      "-language:higherKinds",
+      "-Ypartial-unification"
+    ),
+
     libraryDependencies ++= Seq(
       "ch.qos.logback" % "logback-classic" % LogbackVersion,
       "com.iheart" %% "ficus" % FicusVersion,
       "com.opentable.components" % "otj-pg-embedded" % OtjPgEmbeddedVersion,
       "com.typesafe" % "config" % TypesafeConfigVersion,
       "com.typesafe.scala-logging" %% "scala-logging" % TypsafeLoggingVersion,
       "org.http4s" %% "http4s-blaze-server" % Http4sVersion,
+      "org.http4s" %% "http4s-blaze-client" % Http4sVersion,
       "org.http4s" %% "http4s-circe" % Http4sVersion,
       "org.http4s" %% "http4s-dsl" % Http4sVersion,
       "org.liquibase" % "liquibase-core" % LiquibaseVersion,
       "org.postgresql" % "postgresql" % PostgresqlVersion,
 
       "org.specs2" %% "specs2-core" % Specs2Version % Test,
     ),
+
+    libraryDependencies ++= Seq(
```

Comment:
Might be better to just include this in the block above. That way we have only one way to define dependencies.